R/deparse-data-frame.R

```diff
@@ -0,0 +1,137 @@
+#' @export
+deparse.data.frame <-
+  function(x, as_tibble = FALSE, as_tribble = FALSE, generate_mutate = TRUE, ...) {
```

Comment:
The problem with `as_tribble` is that after the discussion you, I and Hadley had regarding extending the `tribble` syntax, it's relatively harder to use `tribble` for data-types beyond the basic ones. I've got some code which generates a piped call to `mutate` for `factor`s, `Date`s, etc., but it's not ideal.